oceanbasecloud/metadata.csv

```diff
@@ -0,0 +1,16 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+oceanbasecloud.load1,gauge,,,,Past 1 minute system average load,1,oceanbasecloud,,
+oceanbasecloud.cpu_percent,gauge,,percent,,CPU Percent,1,oceanbasecloud,,
+oceanbasecloud.memory_used,gauge,,percent,,Used physical memory percent,1,oceanbasecloud,,
+oceanbasecloud.net_throughput,gauge,,mebibyte,,Network throughput,1,oceanbasecloud,,
+oceanbasecloud.ob_data_disk_percent,gauge,,percent,,OB Data Disk Usag,1,oceanbasecloud,,
+oceanbasecloud.ob_clog_disk_percent,gauge,,percent,,OB Clog Disk Usage,1,oceanbasecloud,,
+oceanbasecloud.ob_process_exists,gauge,,,,OB process exists status,1,oceanbasecloud,,
+oceanbasecloud.sql_all_count,gauge,,execution,,Number of SQL statements processed per second,1,oceanbasecloud,,
+oceanbasecloud.sql_all_rt,gauge,,millisecond,,The average processing time of each SQL statement,1,oceanbasecloud,,
+oceanbasecloud.all_session,gauge,,,,Number of current sessions,1,oceanbasecloud,,
+oceanbasecloud.request_queue_time,gauge,,microsecond,,Request waiting queue time consuming,1,oceanbasecloud,,
```

Comment:
```suggestion
oceanbasecloud.request_queue_time,gauge,,microsecond,,Time spent waiting in the request queue,1,oceanbasecloud,,
```

Reply:
Suggesting a minor clarification, please confirm if this update is still accurate